WordPress/src/main/java/org/wordpress/android/ui/stories/prefs/StoriesPrefs.kt

```diff
@@ -32,13 +35,60 @@ class StoriesPrefs @Inject constructor(
                 KEY_PREFIX_LOCAL_MEDIA_ID + mediaId.value.toString()
     }
 
+    private fun buildSlideKey(siteId: Long, tempId: TempId): String {
+        return KEY_PREFIX_STORIES_SLIDE_ID + siteId.toString() + "-" +
+                KEY_PREFIX_TEMP_MEDIA_ID + tempId.id
+    }
+
+    @SuppressLint("ApplySharedPref")
```

Comment:
Could you expound a bit on why you suppressed using apply here Is it because apply does it in the background and you want this to be done immediately?

Reply:
exactly :) - I need to block the call to make sure it's saved and then we can continue on.
Just in case of curiosity, I've [plans tracked here](https://github.com/Automattic/stories-android/issues/445) to move to [Jetpack Datastore](https://developer.android.com/topic/libraries/architecture/datastore) afterwards - aim is to finish this current implementation for now 👍

Reply:
Okay, awesome. Thank you. Looking forward to seeing Jetpack datastore in action.